docs/source/roles/researcher/using_the_sre.md

```diff
@@ -0,0 +1,744 @@
+In the event of any data loss due to accidental data deletion by a TRE user, your system administrator can restore the `/backup/` folder to the state it was in at an earlier point in time (up to 12 weeks in the past).
+This **cannot** be used to recover individual files - only the complete contents of the folder.
+Everyone in your group will have **read-and-write access** to all folders on `/backup`.
+
+The contents of `/backup/` will be **identical** on all workspaces in your SRE.
+-->
+
+### Output resources
+
+Any outputs that you want to extract from the secure environment should be placed in the **/output/** folder on the workspace.
+
+- The contents of **/output/** will be identical on all workspaces in your SRE.
+- Everyone working on your project will be able to access it
+- Everyone has **read-and-write access** to the files stored here.
+
+Anything placed in here will be considered for data egress - removal from the secure research environment - by the project's principal investigator together with the data provider.
+
+:::{tip}
+You may want to consider having subfolders of **/output/** to make the review of this directory easier.
+:::
+
+## {{pill}} Version control using Gitea
+
+**Gitea**[^footnote-gitea] is an open-source code hosting platform for version control and collaboration - similar to **GitHub**.
+It allows you to use [git](https://git-scm.com/about) to **version control** your work, coordinate tasks using **issues** and review work using **pull requests**.
+
+[^footnote-gitea]: **Gitea** is an open source project. We want to thank the community for maintaining free and open source software for us to use and reuse. You can read more about **Gitea** at [their website](<https://about.gitea.com/>).
+
+The **Gitea** server within the SRE can hold code, documentation and results from your team's analyses.
+Use the **Gitea** server to work collaboratively on code with other project team members.
+
+:::{important}
+This **Gitea** server is entirely within the SRE - you do not need to worry about the security of the information you upload there as it is inaccessible from the public internet.
+:::
+
+You can access **Gitea** from an internet browser in the workspace using the desktop shortcut.
+Use your **{ref}`short-form username <roles_researcher_username>`** and **password** to login.
+
+::::{admonition} Logging in to Gitea
+:class: dropdown note
+
+- Click the **{guilabel}`Sign in`** button on the top-right of the page.
+
+    :::{image} images/gitea_homepage.png
+    :alt: Gitea homepage
+    :align: center
+    :width: 90%
+    :::
+
+- Enter your **{ref}`short-form username <roles_researcher_username>`** and **password**.
+
+    :::{image} images/gitea_login.png
+    :alt: Gitea login
+    :align: center
+    :width: 90%
+    :::
+
+- Then click the **{guilabel}`Sign in`** button
+
+::::
+
+::::{admonition} Create a new repository
+:class: dropdown note
+
+- Log in to the **Gitea** dashboard
```

Comment:
I would drop the Gitea documentation from here on and replace it with the Gitea docs.

Leave it with the specifics about credentials in the TRE and internal-only nature.

Reply:
The Gitea docs are actually pretty poor on most of these things. I'll have a go though.

Reply:
e.g. there's no equivalent of the Github [create a repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-new-repository) page.

Reply:
In that case, lets keep this for now?
Maybe open an issue about finding an alternative.

Reply:
Captured in #2131